src/lib.rs

```diff
@@ -132,6 +140,55 @@ impl File {
         }
     }
 
+    /// Creates a new [`File`] from an existing file on disk.
+    ///
+    /// # Arguments
+    /// - `path` - The path to the file.
+    ///
+    /// # Returns
+    /// - [`File`] - The new File.
+    ///
+    /// # Example
+    /// ```
+    /// let file = piston_rs::File::load_from("src/lib.rs");
+    ///
+    /// assert!(file.content.contains("pub fn load_from"));
+    /// assert_eq!(file.name, "lib.rs".to_string());
+    /// assert_eq!(file.encoding, "utf8".to_string());
+    /// ```
+    pub fn load_from(path: &str) -> Self {
```

Comment:
Rather than making the decision to just print and return a default `File` here, I would prefer if we passed an error back up to the user to handle how they see which (which may or may not include using a default file). 

We will need to use `std::error`, and probably `std::io::Error`, and the appropriate `std::io::ErrorKind` for the errors. 

Alternatively, if you dont want to use the built in errors, you can create a struct to represent a `FileLoadError` that we make. It will need to inplement the `Display` trait if we go this route. 

```suggestion
    pub fn load_from(path: &str) -> Result<Self, Box<dyn error::Error>> {
```